aries-site/src/examples/templates/feedback/PowerOnExample.js

```diff
@@ -0,0 +1,205 @@
+import React, { useCallback, useContext, useEffect, useState } from 'react';
+import {
+  Anchor,
+  Box,
+  Button,
+  Card,
+  Layer,
+  Form,
+  Footer,
+  FormField,
+  Heading,
+  Spinner,
+  StarRating,
+  ResponsiveContext,
+  Text,
+  TextArea,
+} from 'grommet';
+import { StatusGoodSmall } from 'grommet-icons';
+
+const FeedbackLayout = ({ children, onClick, onClose }) => {
+  const breakpoint = useContext(ResponsiveContext);
+  return (
+    // feedback layout
+    <Layer>
+      <Box
+        fill="vertical"
+        overflow="auto"
+        width={!['xsmall', 'small'].includes(breakpoint) ? 'medium' : undefined}
+        pad="medium"
+        gap="medium"
+      >
+        <Heading level={4} size="small" margin="none">
+          Feedback on powering devices
+        </Heading>
+        {children}
+        <Footer direction="row" justify="end" gap="small">
+          <Button onClick={onClose} label="Cancel" />
+          <Button
+            onClick={onClick}
+            label="Submit Feedback"
+            primary
+            type="submit"
+          />
+        </Footer>
+      </Box>
+    </Layer>
+  );
+};
+
+export const PowerOnExample = () => {
+  const [isLoading, setIsLoading] = useState(false);
+  const [isSuccessful, setIsSuccessful] = useState(false);
+  const [rating, setRating] = useState();
+  const [showFeedback, setShowFeedback] = useState(false);
+
+  useEffect(() => {
+    setTimeout(() => {
+      if (isSuccessful) {
+        setShowFeedback(true);
+      }
+    }, 1000);
+  }, [isSuccessful]);
+
+  // simulating a call to a power on API call. Demo purposes
+  const PowerOnLoading = () => {
+    setIsLoading(true);
+    setTimeout(() => {
+      setIsLoading(false);
+      setIsSuccessful(true);
+    }, 3000);
+  };
+
+  // fetch call to POST api should be in onSubmit
+  // for demo purposes disabling actual API call
+  // submitFeedback();
+  const onSubmit = useCallback(value => {
+    const data = {
+      values: {
+        // url for the route which the feedback survey is presented
+        fullURL: url,
+        // type of device. Possible values 'desktop', 'tablet', 'mobile'.
+        deviceType: deviceType,
+        // QIDs are unique to each feedback instance and will be provided when
+        // coordinating with program manager, Doris Singer (doris.singer@hpe.com).
+        QID1: value.value['power-rating-example'],
+        QID2_TEXT: value.value['power-textArea-example'],
+      },
+    };
+    // dev uses POST https://api.qualtrics.com/{survey-id}-create-a-new-response
+    fetch(API_URL, {
+      method: 'POST',
+      headers: {
+        'Content-Type': 'application/json',
+        Accept: 'application/json',
+        'X-API-TOKEN': API_TOKEN,
+      },
+      body: JSON.stringify(data),
+    })
+      .then(response => response.json())
+      .catch(error => {
+        // Error handling here
+      });
+  }, []);
+
+  return (
+    <Card width="large" gap="medium" justify="center" pad="medium">
```

Comment:
Let's connect on this. We should be reusing the [ModalDialog component](https://github.com/grommet/hpe-design-system/tree/master/aries-site/src/examples/templates/ModalDialog) so that we aren't having to re-invent the layout.